### PR TITLE
Add flyways migration

### PR DESCRIPTION
**Pull request by wsargent**
Add database migration link to https://flywaydb.org/

### CODE REVIEW

File: slick/src/sphinx/migrations.rst [hunk 1]
```diff
@@ -26,3 +26,16 @@ slick-migration-api
 library for defining database migrations, for use with Slick. It supports
 altering database schemas using a type-safe Scala DSL. Unfortunately, it
 does not support Slick of version 3.0 or later for the time being.
+
+Flyways
```
Comment (cvogt) [inline slick/src/sphinx/migrations.rst]: Isn't it called Flyway without an s ?

File: slick/src/sphinx/migrations.rst [hunk 2]
```diff
@@ -26,3 +26,16 @@ slick-migration-api
...
+--------------------------
+
+`Flyways <https://flywaydb.org/>`_ is a database migration tool that can
+ run SQL or Java migration against a database.
```
Comment (cvogt) [inline slick/src/sphinx/migrations.rst]: migrations

**Comment by cvogt** [comment]
nice :)!

File: slick/src/sphinx/migrations.rst [hunk 3]
```diff
@@ -26,3 +26,16 @@ slick-migration-api
...
+
+Flyway
+--------------------------
+
+`Flyway <https://flywaydb.org/>`_ is a database migration tool that can
+ run SQL or Java against a database.
+
+`SBT integration <https://flywaydb.org/documentation/sbt/>`_ is available,
+and there is an example `Play project <https://github.com/playframework/play-isolated-slick#database-migration>`_
+that shows Flyways in a module with Slick code generation.
```
Comment (lastland) [inline slick/src/sphinx/migrations.rst]: s/Flyways/Flyway ?

File: slick/src/sphinx/migrations.rst [hunk 4]
```diff
@@ -26,3 +26,16 @@ slick-migration-api
 library for defining database migrations, for use with Slick. It supports
 altering database schemas using a type-safe Scala DSL. Unfortunately, it
 does not support Slick of version 3.0 or later for the time being.
+
+Flyway
+--------------------------
+
+`Flyway <https://flywaydb.org/>`_ is a database migration tool that can
+ run SQL or Java against a database.
```
Comment (d6y) [inline slick/src/sphinx/migrations.rst]: Hard to spot... but there's an extra space at the start of line 34, which causes "run SQL..." to be indented :-)

database migrations slick 3 2 0-snapshot documentation 2016-05-31 12-21-45

Can we remove that space please?

BTW, you can generate the site with `sbt make-site`. The HTML is written to _slick/slick/target/sphinx/docs/index.html_.

**Comment by wsargent** [comment]
Pushed fixed spelling, should be good for review

**Comment by d6y** [comment]
👍 

database migrations slick 3 2 0-snapshot documentation 2016-06-05 11-56-24

**Comment by cvogt** [comment]
Thx @wsargent :) !!

**Comment by nafg** [comment]
I better merge the PR for slick 3 ;)

**Comment by nafg** [comment]
I did (see #1519)

**Comment by lastland** [comment]
@nafg Awesome! I will see how people can use slick-migration-api with Forklift.

**Pull request merged**